cmd/sand-agent/sand-agent.go

```diff
@@ -94,22 +93,36 @@ func main() {
 	nctrl := web.NewNetworksController(c, networkRepository, endpointRepository, ipAllocator)
 	ectrl := web.NewEndpointsController(c, networkRepository, endpointRepository, ipAllocator)
 
-	r := handlers.NewRouter(log)
-	r.Use(handlers.ErrorMiddleware)
-	r.HandleFunc("/version", vctrl.Show).Methods("GET")
-	r.HandleFunc("/networks", nctrl.List).Methods("GET")
-	r.HandleFunc("/networks", nctrl.Create).Methods("POST")
-	r.HandleFunc("/networks/{id}", nctrl.Show).Methods("GET")
-	r.HandleFunc("/networks/{id}", nctrl.Destroy).Methods("DELETE")
-	r.HandleFunc("/networks/{id}", nctrl.Connect).Methods("CONNECT")
-	r.HandleFunc("/endpoints", ectrl.Create).Methods("POST")
-	r.HandleFunc("/endpoints", ectrl.List).Methods("GET")
-	r.HandleFunc("/endpoints/{id}", ectrl.Destroy).Methods("DELETE")
+	sandRouter := handlers.NewRouter(log)
+	sandRouter.Use(handlers.ErrorMiddleware)
+	sandRouter.HandleFunc("/version", vctrl.Show).Methods("GET")
+	sandRouter.HandleFunc("/networks", nctrl.List).Methods("GET")
+	sandRouter.HandleFunc("/networks", nctrl.Create).Methods("POST")
+	sandRouter.HandleFunc("/networks/{id}", nctrl.Show).Methods("GET")
+	sandRouter.HandleFunc("/networks/{id}", nctrl.Destroy).Methods("DELETE")
+	sandRouter.HandleFunc("/networks/{id}", nctrl.Connect).Methods("CONNECT")
+	sandRouter.HandleFunc("/endpoints", ectrl.Create).Methods("POST")
+	sandRouter.HandleFunc("/endpoints", ectrl.List).Methods("GET")
+	sandRouter.HandleFunc("/endpoints/{id}", ectrl.Destroy).Methods("DELETE")
 
 	log.WithField("port", c.HttpPort).Info("Listening")
 	serviceEndpoint := fmt.Sprintf(":%d", c.HttpPort)
 
-	wg := &sync.WaitGroup{}
+	// We can only have one graceful service per process
```

Comment:
suggestion: Would it be possible to add more context it?
```suggestion
	// We can only have one graceful service per process since graceful 1.2.0
```